src/Microsoft.AspNet.Server.Kestrel/compiler/preprocess/SPCM.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Microsoft.AspNet.Server.Kestrel
+{
+    public class StandardsPoliceCompileModule : Microsoft.StandardsPolice.StandardsPoliceCompileModule
```

Comment:
namespace doesn't matter, probably does need to be public, but doesn't get added to the actual output assembly